src/common/helpers/search.js

```diff
@@ -202,7 +202,8 @@ export function autocomplete(orm, query, type) {
 
 	const dslQuery = {
 		body: {
-			query: queryBody
+			query: queryBody,
+			size: 10000
```

Comment:
That looks like a pretty extreme number… I'm not sure what the implications would be in terms of time to run the queries, but let's not overdo it and keep that number  more reasonable.
After all, this autocomplete endpoint sis aimed at humans using the website, and it's unlikely that a human would go through a thousand autocomplete suggestions :)
Let's go with… 42?

Reply:
I have set it to the max limit allowed by the API
Changed the size to 42.